02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,69 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Jake's Webpage</title>
 </head>
 <body>
+  <div align="center">
+    <h1>About Jake!!</h1>
+      <p>Hello!! Jake is the coolest guy around. He somehow manages to work 40 hours at HEB <em><strong>(as a cash controller)</strong></em>, while attending ACA and ACC for a collective 12 more hours per week. Although he is extremely busy, he still finds the time to spend with family and friends.</p>
+  </div>
+  <div>
+    <h1>Jake's Hobbies:</h1>
+      <ul>
+        <li>Coding of course!</li>
+        <li>Playing the piano</li>
+        <li>Getting a nice run in</li>
+        <li>Reading great books as well as listening to great audiobooks</li>
+        <li>Hanging out with much loved family and friends</li>
+        <li>Learning as much as he can everyday- sharpening the saw</li>
+      </ul>
+  </div>
+  <div>
+    <h1>Some items included on Jake's bucket list:</h1>
+      <ol>
+        <li>Skydiving</li>
+        <li>Learning Spanish</li>
+        <li>Traveling all over the World</li>
+      </ol>
+  </div>
+  <div id="favorites" align="center">
+    <section id="fs">
+      <h1>His Favorite T.V. Shows:</h1>
+        <a href="http://www.hbo.com/game-of-thrones">
+          <img src="http://media.moddb.com/images/members/1/123/122021/profile/c9lzmv4d3mgzpnyntz7s.jpg" height="250" width="250"/>
+        </a>
+        <a href="http://www.amc.com/shows/the-walking-dead">
+          <img src="http://cdn.bgr.com/2015/01/the-walking-dead-season-5-trailer.png" height="250" width="250"/>
+        </a>
+        <a href="http://www.sho.com/dexter">
+          <img src="https://secure.netflix.com/us/boxshots/tv_sdp_s/70136126.jpg" height="250" width="250" />
+        </a>
+        <a href="http://www.amc.com/shows/breaking-bad">
+          <img src="http://unspoiledpodcasts.com/wp-content/uploads/2014/09/Breaking-Bad.jpg" height="250" width="250"/>
+        </a>
+    <section id="fb">
```

Comment:
After closing `a` tag, the following opening element needs to be on the same level. So either your indentation is wrong somewhere or you are missing a closing tag.